app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+
+  def base_container
+    if controller.controller_name == "movies"
+      return "video-base-container"
```

Comment:
Rubyでは最後が戻り値となります。`return`は書かないのが一般的です。
```suggestion
      "video-base-container"
```